src/dlgProfilePreferences.cpp

```diff
@@ -1457,8 +1457,7 @@ void dlgProfilePreferences::setDisplayFont()
     }
     QFont font = fontComboBox->currentFont();
     font.setPointSize(mFontSize);
-    if (pHost->mDisplayFont != font) {
-        pHost->mDisplayFont = font;
+    if (pHost->getDisplayFont() != font && pHost->setDisplayFont(font)) {
```

Comment:
If the user selects such a 0 width font now, nothing happens when they click save. No message or error. They will wonder why.